test/unit/ard/cost/test_wisdem_wrap.py

```diff
@@ -73,8 +69,8 @@ def test_baseline_farm(self):
         self.prob.run_model()
 
         # use a file of pyrite-standard data to validate against
-        fn_pyrite = os.path.join(
-            os.path.split(__file__)[0],
+        fn_pyrite = Path(
+            Path(__file__).parent,
```

Comment:
Here you could do:

```python
fn_pyrite = Path(__file__).parent / "test_landbosse_wrap_baseline_farm.npz"
```

The `/` (`__truediv__`) function is overloaded on Path, and just like for a number it returns a new object of the same type.